modules/kubernetes-addons/cluster-autoscaler/locals.tf

```diff
@@ -36,7 +36,7 @@ locals {
 
   irsa_config = {
     create_kubernetes_namespace       = false
-    kubernetes_namespace              = "kube-system"
```

Comment:
Here we are forcing the Cluster CA to run on `kube-ssytem`.

Running CA in other namespace on a user node is risky. If CA erases the node it is sitting on and some race condition occurs you may end up with cluster without both CA and capacity to run anything. 

Hence its really important that...

CA stays in `kube-system` OR it has restrictive PodDisruptionBudget
CA is run at high priority or is marked as a critical pod